### DISCUSSION TITLE
How can i modify the ticket nameslicing in the /tickets website endpoint?

### DISCUSSION THREAD

**Discussion by rodrigolfh** [django-helpdesk/django-helpdesk#1226] [Q&A] [answered]
in the /tickets the ticket title is limited to about 25 characters but as it is so little you don't get the idea of what the ticket is about.
I assume there is some kind of slicing but couldn't find where is the code located.
![image](https://github.com/user-attachments/assets/8e370adb-0261-4940-a064-08b66779edb3)

**Comment by timthelion** [comment]
I think you want https://github.com/django-helpdesk/django-helpdesk/blob/cc6e5978407fe0f86dd990d440766aea20363bab/helpdesk/static/helpdesk/helpdesk-extend.css#L104

**Reply by timthelion** [reply, marked as answer]
Try just overriding that css in your browser dev tools to see if it helps.

**Reply by rodrigolfh** [reply]
It works!! Thanks!
